### PR TITLE
feat: Develop firstName and lastName fields

### PR DESCRIPTION
**Pull request by piotr-pajak**
## Jira issue(s)
[LC-260](https://selleolabs.atlassian.net/browse/LC-260)

## Overview
Added first name and last name fields with validation.
Updated `Register.page.test.tsx` to include tests for the new fields.

## Screenshots
Screenshot 2024-10-02 at 14 24 21
Screenshot 2024-10-02 at 14 24 32

[LC-260]: https://selleolabs.atlassian.net/browse/LC-260?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: apps/web/app/modules/Auth/Register.page.test.tsx [hunk 1]
```diff
@@ -29,14 +29,16 @@ describe("Register page", () => {
...
-      screen.getByRole("heading", { name: "Sign Up" })
+      screen.getByRole("heading", { name: "Sign Up" }),
     ).toBeInTheDocument();
   });
 
   it("submits the form with valid data", async () => {
     renderWith({ withQuery: true }).render(<RemixStub />);
 
     const user = userEvent.setup();
+    await user.type(screen.getByLabelText("firstName"), "Name");
```
Comment (typeWolffo) [inline apps/web/app/modules/Auth/Register.page.test.tsx:40]:
```suggestion
    await user.type(screen.getByLabelText("First name"), "Name");
```

File: apps/web/app/modules/Auth/Register.page.test.tsx [hunk 2]
```diff
@@ -29,14 +29,16 @@ describe("Register page", () => {
...
+    await user.type(screen.getByLabelText("lastName"), "Surname");
```
Comment (typeWolffo) [inline apps/web/app/modules/Auth/Register.page.test.tsx:41]:
```suggestion
    await user.type(screen.getByLabelText("Lastn name"), "Surname");
```
as well

File: apps/web/app/modules/Auth/Register.page.test.tsx (hunk 1 above)
Comment (piotr-pajak) [reply apps/web/app/modules/Auth/Register.page.test.tsx]: Oops, I noticed the mistake, but for some reason, I didn't push the changes.

Comment (piotr-pajak) [reply apps/web/app/modules/Auth/Register.page.test.tsx]: Already fixed ✅

**Review by typeWolffo — APPROVED** [review]

**Pull request merged**